### PR TITLE
Xrddev 371

### PR DESCRIPTION
**Pull request by jansu76**
Builds on XRDDEV-382: https://github.com/nordic-institute/X-Road-REST-UI/pull/12
**Do not merge** before PR 12

### CODE REVIEW

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/MyHibernatePropertiesCustomizer.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * x
```
Comment (TJaakkola) [inline src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/MyHibernatePropertiesCustomizer.java]: Missing/extra comment

File: src/proxy-ui-api/build.gradle [hunk 2]
```diff
@@ -128,18 +131,25 @@ licenseMain.enabled = false
 licenseFormat.dependsOn licenseFormatJava
 licenseTest.dependsOn licenseTestJava
 
+// TODO: would rather use hsqldb (why? maybe since old code uses that)
```
Comment (TJaakkola) [inline src/proxy-ui-api/build.gradle]: Uncleaned comments / TODOs

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/ClientsApiController.java [hunk 3]
```diff
@@ -92,11 +90,30 @@ public RestNotFoundException(String s) {
         return null;
     }
 
+    /**
```
Comment (TJaakkola) [inline src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/ClientsApiController.java]: Are test transactions supposed to go in production code?

File: src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/ClientsApiControllerIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+    @Before
+    public void setup() {
+        when(globalConfWrapper.getMemberName(any())).thenReturn("test-member-name");
+    }
+
+    @Autowired
+    private ClientsApiController clientsApiController;
+
+    // tests with TestRestTemplate would be good, but require some extra work
```
Comment (TJaakkola) [inline src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/ClientsApiControllerIntegrationTest.java]: Are tests supposed to be commented out

Comment (jansu76) [reply src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/ClientsApiControllerIntegrationTest.java]: Yes, this was a "todo later" reminder. Cleaned that out and created https://jira.niis.org/browse/XRDDEV-416

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/MyHibernatePropertiesCustomizer.java (hunk 1 above)
Comment (jansu76) [reply src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/MyHibernatePropertiesCustomizer.java]: fixed

File: src/proxy-ui-api/build.gradle (hunk 2 above)
Comment (jansu76) [reply src/proxy-ui-api/build.gradle]: fixed

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/ClientsApiController.java (hunk 3 above)
Comment (jansu76) [reply src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/ClientsApiController.java]: fixed

**Pull request merged**